nominal/nominal.py

```diff
@@ -339,3 +348,33 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+
+def draft_checklist(
```

Comment:
Open to other naming options .. hoping to not overload the term "draft", which is used on the Front-end of the app, but I'm not actually sure how that maps to the concept of branches/VCS on the backend - we don't use the term draft in the APIs